external-crates/move/crates/move-compiler/src/sui_mode/linters/mod.rs

```diff
@@ -68,6 +69,11 @@ pub const CUSTOM_STATE_CHANGE_FILTER_NAME: &str = "custom_state_change";
 pub const COIN_FIELD_FILTER_NAME: &str = "coin_field";
 pub const FREEZE_WRAPPED_FILTER_NAME: &str = "freeze_wrapped";
 pub const COLLECTION_EQUALITY_FILTER_NAME: &str = "collection_equality";
+pub const RANDOM_OBJECTS_FILTER_NAME: &str = "random_objects";
```

Comment:
Maybe 
```suggestion
pub const RANDOM_OBJECTS_FILTER_NAME: &str = "public_random";
```
Or something similar?

Reply:
Also, the convention is to name the the source file containing the linter consistently with the linter itself (it would be `public_random.rs` after this suggested change then).